rasa/core/channels/botframework.py

```diff
@@ -215,6 +262,36 @@ async def health(request: Request) -> HTTPResponse:
 
         @botframework_webhook.route("/webhook", methods=["POST"])
         async def webhook(request: Request) -> HTTPResponse:
+            # Update the JWT keys daily
```

Comment:
I suggest extracting this new code (L265-294) into a public method that should be unit tested too.